<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fx</class>
 <widget class="QWidget" name="fx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>604</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>时间范围：2015/1/1--2016/2/12</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>开始时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2015</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="maximumDate">
          <date>
           <year>2016</year>
           <month>2</month>
           <day>12</day>
          </date>
         </property>
         <property name="minimumDate">
          <date>
           <year>2015</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>结束时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2015</year>
           <month>9</month>
           <day>14</day>
          </datetime>
         </property>
         <property name="maximumDateTime">
          <datetime>
           <hour>15</hour>
           <minute>59</minute>
           <second>59</second>
           <year>2016</year>
           <month>2</month>
           <day>12</day>
          </datetime>
         </property>
         <property name="minimumDate">
          <date>
           <year>2015</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="rb3">
         <property name="text">
          <string>站点客流量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbh">
         <property name="text">
          <string>后移</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="rb2">
         <property name="text">
          <string>川渝双向客流量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbq">
         <property name="text">
          <string>前移</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QRadioButton" name="rb1">
         <property name="text">
          <string>列车载客率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbs">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QChartView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
